### DISCUSSION TITLE
南极谋杀案（十一）- 时空黑客 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#441] [General]
# 南极谋杀案（十一）- 时空黑客 - Miao Yu | 于淼

https://yufree.cn/cn/2023/10/22/antarictic-murder/

**Comment by tcgriffith** [comment]
现在有seedance2了，老于要不要试试影视化一下 ;)

**Reply by yufree** [reply]
还没计划，这玩意连个主人公都没有，更多是个设定集，科幻作品要合理化特定情节得补大量设定，现在AI上下文不一定撑得住，文生视频适合背景设定不复杂但故事明确的作品，这个基本没故事。
